### PR TITLE
Adding the class for creating a new console command.

### PR DESCRIPTION
**Pull request by eduard13**
## Purpose of this pull request

This pull request (PR) adds the required class for creating and running a new console command. Also provides some examples how to style the command output, by showing an error message or a success one.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.1/extension-dev-guide/cli-cmds/cli-howto.html
- https://devdocs.magento.com/guides/v2.2/extension-dev-guide/cli-cmds/cli-howto.html
- https://devdocs.magento.com/guides/v2.3/extension-dev-guide/cli-cmds/cli-howto.html

## Links to Magento source code

- https://github.com/magento/magento2/blob/2.3/app/code/Magento/Store/Console/Command/StoreListCommand.php

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.1/extension-dev-guide/cli-cmds/cli-howto.md [hunk 1]
```diff
@@ -35,20 +35,63 @@ Following is a summary of the process:
...
+            $this->setDescription('This is my first console command.');
+            
+            parent::configure();
+        }
+    
+        /**
+         * @param InputInterface $input
+         * @param OutputInterface $output
+         *
+         * @return void
```
Comment (rogyar) [inline guides/v2.1/extension-dev-guide/cli-cmds/cli-howto.md]: Please, change the return type to `int|null` according to the parent class.

Comment (eduard13) [reply guides/v2.1/extension-dev-guide/cli-cmds/cli-howto.md]: Thank you, the return type has been adjusted.

**Comment by jeff-matthews** [comment]
running tests

**Comment by dshevtsov** [comment]
@jeff-matthews, please add `whatsnew`.

**Comment by jeff-matthews** [comment]
@dshevtsov, done

**Pull request merged**